Make it work on Windows

Continuing the theme of #724... the global install does not work on Windows without WSL. Node seems to be passing stuff into cmd.

<details>
<summary>error message</summary>

```
> auspice@2.3.7 postinstall C:\Users\eleth\AppData\Roaming\npm\node_modules\auspice
> npm run create-data-dir


> auspice@2.3.7 create-data-dir C:\Users\eleth\AppData\Roaming\npm\node_modules\auspice
> ./scripts/create-data-dir.sh

'.' 不是内部或外部命令，也不是可运行的程序或批处理文件。
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! auspice@2.3.7 create-data-dir: `./scripts/create-data-dir.sh`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the auspice@2.3.7 create-data-dir script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\eleth\AppData\Roaming\npm-cache\_logs\2020-01-30T06_33_48_240Z-debug.log
npm WARN notsup Unsupported engine for auspice@2.3.7: wanted: {"node":"10.8.x","npm":"6.2.x"} (current: {"node":"13.3.0","npm":"6.13
.1"})
npm WARN notsup Not compatible with your version of node/npm: auspice@2.3.7
npm WARN babel-eslint@10.0.3 requires a peer of eslint@>= 4.12.1 but none is installed. You must install peer dependencies yourself.
npm WARN react-hot-loader@4.12.19 requires a peer of @types/react@^15.0.0 || ^16.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN react-addons-css-transition-group@15.6.2 requires a peer of react@^15.4.2 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.11 (node_modules\auspice\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.11: wanted {"os":"darwin","arch":"any"} (current
: {"os":"win32","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! auspice@2.3.7 postinstall: `npm run create-data-dir`
npm ERR! Exit status 1
```

</details>

(Yeah I know the error message is all chinese to you. It says '.' is not a command, an executable or a batch file.)

To fix this issue we can probably put "bash" in front of these commands so we are explicitly calling it with bash. This does bring out the additional problem of WSL shipping a binary called "bash" that never errors out. To fix this we can use "env bash" instead -- Cygwin/MSYS always ship with `env`, and they are 90% percent of the cases where people have a proper bash on Windows anyways.

--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
   },
   "main": "index.js",
   "scripts": {
-    "create-data-dir": "./scripts/create-data-dir.sh",
+    "create-data-dir": "env bash ./scripts/create-data-dir.sh",
     "postinstall": "npm run create-data-dir",
     "view": "node auspice.js view --verbose",
     "dev": "node auspice.js develop --verbose",
@@ -24,11 +24,11 @@
     "build": "node auspice.js build --verbose",
     "prepare": "npm run build",
     "lint": "eslint src",
-    "get-data": "./scripts/get-data.sh",
-    "get-narratives": "./scripts/get-narratives.sh",
+    "get-data": "env bash ./scripts/get-data.sh",
+    "get-narratives": "env bash ./scripts/get-narratives.sh",
     "heroku-postbuild": "npm run build && npm run get-data && npm run get-narratives",
-    "rebuild-docker-image": "./scripts/rebuild-docker-image.sh",
-    "gzip-and-upload": "./scripts/gzip-and-upload.sh",
+    "rebuild-docker-image": "env bash ./scripts/rebuild-docker-image.sh",
+    "gzip-and-upload": "env bash ./scripts/gzip-and-upload.sh",
     "build-docs": "echo 'see ./docs-src/README.md'"
   },
   "dependencies": {

